### ISSUE TITLE
When using some Chinese fonts, the text is displayed as gibberish

### ISSUE THREAD

**Issue by liweihao5253** [iina/iina#1417]
- macOS 10.13.2
- IINA 0.0.15
When the " ASS " subtitle file contains the font style shown in the song body, or in IINA settings where the font is set to the song body - Jane or the Chinese song body, IINA is displayed as a mess, as shown in the figure below:
![35099037-7ce26d0c-fc91-11e7-8ead-2004143ca16b](https://user-images.githubusercontent.com/35565160/35370274-b700f766-01c7-11e8-8fef-0d988009d7f4.png)
This is the correct display in MPV:
![35099081-a21fbf8e-fc91-11e7-8287-8cf99884dc39](https://user-images.githubusercontent.com/35565160/35370298-d8d42dfe-01c7-11e8-8c57-4e9d35eb5898.png)

1.No matter what subtitling file, the choice of song body - Jane or Chinese song body in IINA settings can be displayed as gibberish.
2.Instead of using IINA settings, using the settings inside the ass file, either " STSong " or " song body - Jane " and " Chinese song body " can also be displayed as gibberish.
3.None of the other system fonts found this problem.

The bottom line is that no matter what route you take or how you use it ( UTF-8, UTF-16, GBK all tried ) once IINA has adopted the system's native Chinese song and song dynasty, Jane, it will be a mess.
None of the other players will, so it shouldn't be a problem with my character set.

**Comment by inflation** [comment]
I’m suspecting you’re talking about font names, so next time if you’re not sure about their English names, just use the original ones.  

Have you tried other fonts? It’s weird since IINA and mpv both use libass. 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>
